app/src/main/java/com/example/dennis/leagueoflegions/Army.java

```diff
@@ -1,7 +1,14 @@
 package com.example.dennis.leagueoflegions;
 
 public class Army extends Unit {
+    private final static String type = "circle";
+    
```

Comment:
this should be:
private static final String TYPE = "army";
(capital TYPE, "army" so we can use this string for other purposes)